lib/select.ts

```diff
@@ -205,6 +207,14 @@ export function getOptionFull<T>(option: IOption<T>): IOptionFull<T> {
   return (typeof option === "string") ? {value: option, label: option} : (option as IOptionFull<T>);
 }
 
+// Prevents select button text overflow - assumes buttonArrow width is ~20% of button.
+const cssBtnText = styled('div', `
+  width: 80%;
```

Comment:
Since it's part of a flexbox, might it be cleaner to omit the width, but instead wrap `options.buttonArrow` into an element with `flex: none`?